pyvelox/pyvelox.h

```diff
@@ -109,6 +111,27 @@ inline velox::variant pyToVariant(const py::handle& obj, const Type& dtype) {
   }
 }
 
+inline void checkRowVectorBounds(const RowVectorPtr& v, vector_size_t idx) {
+  if (idx < 0 || size_t(idx) >= v->childrenSize()) {
+    throw std::out_of_range("Index out of range");
+  }
+}
+
+inline VectorPtr getVectorFromRowVectorPtr(
+    const RowVectorPtr& v,
+    vector_size_t idx) {
+  checkRowVectorBounds(v, idx);
+  return v->childAt(idx);
+}
+
+inline std::string rowVectorToString(const RowVectorPtr& v) {
+  if (v->childrenSize() > 0) {
+    return v->toString(0, v->size());
```

Comment:
why not just return v->toString(0, v->size()) ? It should work even if there are no children..